lib/passageidentity/auth.rb

```diff
@@ -221,11 +234,11 @@ def user_exists?(user_id)
     end
 
     def get_cache(key)
-      @app_cache[key]
+      @app_cache.read(key)
     end
 
-    def set_cache(key, value)
-      @app_cache[key] = value
+    def set_cache(key:, jwks:)
+      @app_cache.write(key, jwks, expires_in: 3600)
```

Comment:
This can be a little more lax. Our node SDK expires at [24 hours](https://github.com/passageidentity/passage-node/blob/c94769c59e28139d7546cd0335dc4bf4476ccf50/src/classes/Auth/Auth.ts#L29)